### ISSUE TITLE
Pylance reports errors in EJS file

### ISSUE THREAD

**Issue by nwalters512** [microsoft/pylance-release#4591]
## Environment data

-   Language Server version: 2023.7.10
-   OS and version: macOS 13.4.1
-   Python version: 3.10.12

## Repro Steps

Create a directory with an `index.ejs` file:

```sh
mkdir reproduction
cd reproduction
touch index.ejs
```

Open this directory in VSCode:

```sh
code .
```

Place the following contents in `index.ejs`:

```ejs
<!doctype html>
<html lang="en">
  <head>
    <title>Hello, world.</title>
  </head>
</html>
```

Observe that the editor does not contain any errors.

Create a `main.py` file and open it in the editor. Switch back to the `index.ejs` file and wait a few seconds; you may also need to reload the editor.

## Expected behavior

Pylance should not show any errors on an EJS file.

## Actual behavior

Pylance does indeed show irrelevant errors on an EJS file:

<img width="286" alt="Screenshot 2023-07-10 at 16 08 46" src="https://github.com/microsoft/pylance-release/assets/1476544/7a7820d5-5238-4382-a9bb-dfe936dff6ac">

## Logs

```
2023-07-10 16:11:30.017 [info] [Info  - 4:11:30 PM] (66051) Pylance language server 2023.7.10 (pyright 74d8f3c4) starting
2023-07-10 16:11:30.017 [info] [Info  - 4:11:30 PM] (66051) Server root directory: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist
2023-07-10 16:11:30.021 [info] [Info  - 4:11:30 PM] (66051) Starting service instance "pylance-repro"
2023-07-10 16:11:30.047 [info] (66051) No configuration file found.
2023-07-10 16:11:30.047 [info] (66051) No pyproject.toml file found.
2023-07-10 16:11:30.049 [info] [Warn  - 4:11:30 PM] (66051) stubPath /Users/nathan/Desktop/pylance-repro/typings is not a valid directory.
2023-07-10 16:11:30.049 [info] (66051) Assuming Python platform Darwin
2023-07-10 16:11:30.100 [info] [Info  - 4:11:30 PM] (66051) Search paths for /Users/nathan/Desktop/pylance-repro
2023-07-10 16:11:30.100 [info] [Info  - 4:11:30 PM] (66051)   /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib
2023-07-10 16:11:30.100 [info] [Info  - 4:11:30 PM] (66051)   /Users/nathan/Desktop/pylance-repro
2023-07-10 16:11:30.100 [info] [Info  - 4:11:30 PM] (66051)   /Users/nathan/Desktop/pylance-repro/typings
2023-07-10 16:11:30.100 [info] [Info  - 4:11:30 PM] (66051)   /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stubs/...
2023-07-10 16:11:30.100 [info] [Info  - 4:11:30 PM] (66051)   /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs
2023-07-10 16:11:30.100 [info] [Info  - 4:11:30 PM] (66051)   /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10
2023-07-10 16:11:30.100 [info] [Info  - 4:11:30 PM] (66051)   /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib-dynload
2023-07-10 16:11:30.101 [info] [Info  - 4:11:30 PM] (66051)   /opt/homebrew/lib/python3.10/site-packages
2023-07-10 16:11:30.101 [info] (66051) Searching for source files
2023-07-10 16:11:30.102 [info] [Info  - 4:11:30 PM] (66051) Found 1 source file
2023-07-10 16:11:30.133 [info] (66051) [FG] parsing: /Users/nathan/Desktop/pylance-repro/index.ejs (23ms)
2023-07-10 16:11:30.182 [info] (66051) [FG] parsing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/builtins.pyi [fs read 2ms] (48ms)
2023-07-10 16:11:30.200 [info] (66051) [FG] binding: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/builtins.pyi (17ms)
2023-07-10 16:11:30.200 [info] (66051) [FG] binding: /Users/nathan/Desktop/pylance-repro/index.ejs (1ms)
2023-07-10 16:11:30.269 [info] [Info  - 4:11:30 PM] (66051) Background analysis(1) root directory: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist
2023-07-10 16:11:30.270 [info] [Info  - 4:11:30 PM] (66051) Background analysis(1) started
2023-07-10 16:11:30.270 [info] (66051) Background analysis message: setConfigOptions
2023-07-10 16:11:30.271 [info] (66051) Background analysis message: setImportResolver
2023-07-10 16:11:30.271 [info] (66051) Background analysis message: ensurePartialStubPackages
2023-07-10 16:11:30.325 [info] (66051) Background analysis message: setConfigOptions
2023-07-10 16:11:30.325 [info] (66051) Background analysis message: setTrackedFiles
2023-07-10 16:11:30.331 [info] (66051) Background analysis message: markAllFilesDirty
2023-07-10 16:11:30.331 [info] (66051) Background analysis message: setFileOpened
2023-07-10 16:11:30.331 [info] (66051) Background analysis message: analyze
2023-07-10 16:11:30.360 [info] (66051) [BG(1)] analyzing: /Users/nathan/Desktop/pylance-repro/index.ejs ...
2023-07-10 16:11:30.360 [info] (66051) [BG(1)]   parsing: /Users/nathan/Desktop/pylance-repro/index.ejs (27ms)
2023-07-10 16:11:30.360 [info] [Info  - 4:11:30 PM] (66051) Indexer background runner(2) root directory: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist (index)
2023-07-10 16:11:30.361 [info] [Info  - 4:11:30 PM] (66051) Indexing(2) started
2023-07-10 16:11:30.399 [info] (66051) [BG(1)]   parsing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/builtins.pyi [fs read 2ms] (38ms)
2023-07-10 16:11:30.405 [info] (66051) [IDX(2)] scan packages /Users/nathan/Desktop/pylance-repro ...
2023-07-10 16:11:30.406 [info] (66051) [IDX(2)]   read stdlib indices (43ms)
2023-07-10 16:11:30.421 [info] (66051) [BG(1)]   binding: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/builtins.pyi (22ms)
2023-07-10 16:11:30.422 [info] (66051) [BG(1)]   binding: /Users/nathan/Desktop/pylance-repro/index.ejs (0ms)
2023-07-10 16:11:30.435 [info] (66051) [BG(1)]   checking: /Users/nathan/Desktop/pylance-repro/index.ejs ...
2023-07-10 16:11:30.435 [info] (66051) [BG(1)]     parsing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/typing.pyi [fs read 0ms] (9ms)
2023-07-10 16:11:30.439 [info] (66051) [BG(1)]     binding: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/typing.pyi (4ms)
2023-07-10 16:11:30.443 [info] (66051) [BG(1)]     parsing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/typing_extensions.pyi [fs read 1ms] (3ms)
2023-07-10 16:11:30.444 [info] (66051) [BG(1)]     binding: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/typing_extensions.pyi (1ms)
2023-07-10 16:11:30.455 [info] (66051) [BG(1)]     parsing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/_typeshed/__init__.pyi [fs read 0ms] (2ms)
2023-07-10 16:11:30.456 [info] (66051) [BG(1)]     binding: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/_typeshed/__init__.pyi (1ms)
2023-07-10 16:11:30.465 [info] (66051) [BG(1)]     parsing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/types.pyi [fs read 0ms] (9ms)
2023-07-10 16:11:30.467 [info] (66051) [BG(1)]     binding: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/types.pyi (2ms)
2023-07-10 16:11:30.473 [info] (66051) [BG(1)]     parsing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/abc.pyi [fs read 0ms] (1ms)
2023-07-10 16:11:30.474 [info] (66051) [BG(1)]     binding: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/typeshed-fallback/stdlib/abc.pyi (0ms)
2023-07-10 16:11:30.475 [info] (66051) [BG(1)]   checking: /Users/nathan/Desktop/pylance-repro/index.ejs (54ms)
2023-07-10 16:11:30.475 [info] (66051) [BG(1)] analyzing: /Users/nathan/Desktop/pylance-repro/index.ejs (143ms)
2023-07-10 16:11:30.477 [info] (66051) Background analysis message: resumeAnalysis
2023-07-10 16:11:30.478 [info] (66051) [BG(1)] indexing: /Users/nathan/Desktop/pylance-repro/main.py ...
2023-07-10 16:11:30.478 [info] (66051) [BG(1)]   parsing: /Users/nathan/Desktop/pylance-repro/main.py [fs read 0ms] (1ms)
2023-07-10 16:11:30.478 [info] (66051) [BG(1)]   binding: /Users/nathan/Desktop/pylance-repro/main.py (0ms)
2023-07-10 16:11:30.479 [info] (66051) [BG(1)] indexing: /Users/nathan/Desktop/pylance-repro/main.py [found 0] (1ms)
2023-07-10 16:11:30.479 [info] (66051) Indexing Done: /Users/nathan/Desktop/pylance-repro/main.py
2023-07-10 16:11:30.666 [info] (66051) [IDX(2)] scan packages /Users/nathan/Desktop/pylance-repro (304ms)
2023-07-10 16:11:30.666 [info] [Info  - 4:11:30 PM] (66051) scanned(2) 267 files over 1 exec env
2023-07-10 16:11:30.715 [info] (66051) [IDX(2)] index packages /Users/nathan/Desktop/pylance-repro ...
2023-07-10 16:11:30.715 [info] (66051) [IDX(2)]   index execution environment /Users/nathan/Desktop/pylance-repro ...
2023-07-10 16:11:30.715 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_afm.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.715 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_c_internal_utils.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.715 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_cm.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.715 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_color_data.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_constrained_layout.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_docstring.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_enums.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_fontconfig_pattern.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_layoutgrid.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_mathtext.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_mathtext_data.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_pylab_helpers.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_text_helpers.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_tight_bbox.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_tight_layout.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/matplotlib/_version.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/sklearn/_config.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.716 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/sklearn/_distributor_init.pyi [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.717 [info] (66051) [IDX(2)]     indexing: /Users/nathan/.vscode/extensions/ms-python.vscode-pylance-2023.7.10/dist/bundled/stubs/sklearn/_min_dependencies.pyi [skipped: private or protected file] (1ms)
2023-07-10 16:11:30.717 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/concurrent/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.717 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/distutils/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.717 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/idlelib/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/lib2to3/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/pydoc_data/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/test/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/turtledemo/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/wsgiref/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/Cellar/python@3.10/3.10.12_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/xmlrpc/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/Cython/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/alabaster/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/blackd/__init__.py [skipped: no '__all__' defined] (1ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/blib2to3/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/et_xmlfile/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/flake8/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/flake8p/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/material/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/materialx/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/_animation_data.py [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/_blocking_input.py [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/_cm_listed.py [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/_internal_utils.py [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.718 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/_type1font.py [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/pylab.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/sphinxext/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/tests/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/matplotlib/tight_bbox.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/mdx_truly_sane_lists/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/mkdocs/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/mpmath/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/nltk/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pathspec/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pgsanity/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pulp/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pydash/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pyquaternion/__init__.py [skipped: no '__all__' defined] (1ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pytest_cov/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/pyximport/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/recommonmark/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/rpy2/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/scipy/_distributor_init.py [skipped: private or protected file] (0ms)
2023-07-10 16:11:30.719 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/scipy/conftest.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.720 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/scipy/version.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.720 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/sphinx_markdown_builder/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.720 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/text_unidecode/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.720 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/tzdata/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.720 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/watchdog/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.720 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/wheel/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.720 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/yapf/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.720 [info] (66051) [IDX(2)]     indexing: /opt/homebrew/lib/python3.10/site-packages/yapftests/__init__.py [skipped: no '__all__' defined] (0ms)
2023-07-10 16:11:30.726 [info] (66051) [IDX(2)]   index execution environment /Users/nathan/Desktop/pylance-repro [found 5258 in 197 files] (16ms)
2023-07-10 16:11:30.726 [info] (66051) [IDX(2)] index packages /Users/nathan/Desktop/pylance-repro [found 5258 in 1 exec envs] (31ms)
2023-07-10 16:11:30.727 [info] [Info  - 4:11:30 PM] (66051) indexed(2) 197 files over 1 exec env
2023-07-10 16:11:30.738 [info] [Info  - 4:11:30 PM] (66051) Indexing finished(2).
```

## Additional information

I also have an extension installed for EJS language support: https://marketplace.visualstudio.com/items?itemName=DigitalBrainstem.javascript-ejs-support. Disabling this extension had no effect on Pylance incorrectly reporting errors in this file.

This started happening quite recently. I wasn't able to find a reference to this in any other issues, and I also couldn't find many references to it on the internet, though I did find this post on StackOverflow: https://stackoverflow.com/questions/76629174/ejs-file-full-of-pylance-errors

**Comment by heejaechang** [comment]
`{ scheme: 'file', language: 'html' }` added for `jinja` exp is causing all `html` document type to be passed to us. we should either define our own `jinja` document type with `html\htm` file extensions or explicitly check `py/pyi/html/htm` in server side.

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.7.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023731-19-July-2023-PreRelease)
